.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,85 @@
+name: Bug Report
+description: File a bug report to help us improve
+labels:
+  - bug
+body:
+  - type: markdown
+    attributes:
+      value: "# Please search the issue tracker for existing bug reports before submitting your own."
```

Comment:
Would this be better as a checkbox at the end of the form that the user has to tick before they can submit it?

Reply:
I hadn't thought of that, but you're right, that's a good idea.